packages/favorites-dialog/src/visTypes.js

```diff
@@ -17,20 +16,38 @@ import PivotTableIcon from './icons/PivotTableIcon';
 export const CHART = 'CHART';
 export const PIVOT_TABLE = 'PIVOT_TABLE';
 
-const visTypeMap = {
-    COLUMN: { label: i18n.t('Column'), icon: ColumnIcon },
-    STACKED_COLUMN: { label: i18n.t('Stacked column'), icon: StackedColumnIcon },
-    BAR: { label: i18n.t('Bar'), icon: BarIcon },
-    STACKED_BAR: { label: i18n.t('Stacked bar'), icon: StackedBarIcon },
-    LINE: { label: i18n.t('Line'), icon: LineIcon },
-    AREA: { label: i18n.t('Area'), icon: AreaIcon },
-    PIE: { label: i18n.t('Pie'), icon: PieIcon },
-    RADAR: { label: i18n.t('Radar'), icon: RadarIcon },
-    GAUGE: { label: i18n.t('Gauge'), icon: GaugeIcon },
-    YEAR_OVER_YEAR_LINE: { label: i18n.t('Year over year (line)'), icon: YearOverYearLineIcon },
-    YEAR_OVER_YEAR_COLUMN: { label: i18n.t('Year over year (column)'), icon: YearOverYearColumnIcon },
-    SINGLE_VALUE: { label: i18n.t('Single value'), icon: SingleValueIcon },
-    [PIVOT_TABLE]: { label: i18n.t('Pivot table'), icon: PivotTableIcon },
+export const visTypeIcons = {
+    COLUMN: ColumnIcon,
+    STACKED_COLUMN: StackedColumnIcon,
+    BAR: BarIcon,
+    STACKED_BAR: StackedBarIcon,
+    LINE: LineIcon,
+    AREA: AreaIcon,
+    PIE: PieIcon,
+    RADAR: RadarIcon,
+    GAUGE: GaugeIcon,
+    YEAR_OVER_YEAR_LINE: YearOverYearLineIcon,
+    YEAR_OVER_YEAR_COLUMN: YearOverYearColumnIcon,
+    SINGLE_VALUE: SingleValueIcon,
+    [PIVOT_TABLE]: PivotTableIcon
 };
 
-export default visTypeMap;
+export const getVisTypeLabel = type => {
+    const visTypeLabels = {
```

Comment:
One reason not to keep duplicates of these is that the translation needs to be duplicated as well? (because they're in different apps and not consolidated as one translation? Or does transifex merge all app translations to one?). E.g. https://github.com/dhis2/d2-ui/blob/522f35a43867f1488771710cebc6922d0b28d09e/packages/favorites-dialog/i18n/en.pot, where we run the risk of translations not being consistent (e.g. if you translate `Line` in one way here and `Line` in another way here), which would affect the UX greatly.

Reply:
@janhenrikoverland , @amcgee

Reply:
I would rather take this in a separate discussion and implementation. I want to be careful to not reintroduce a potential circular dependency between d2-ui and analytics, so yeah, we need a bigger architectural discussion around this. Also, some of these components will probably be getting migrated to ui-widgets, so that is another thing to consider.

Reply:
@martinkrulltott Transifex has "translation memory" which recommends existing translation matches when translating new strings, so this shouldn't be a HUGE issue, but it is possible that the translator would not accept the suggestion

Reply:
Fair enough 🙂 👍